src/main/app/case/case.ts

```diff
@@ -73,7 +74,9 @@ export const formFieldsToCaseMapping: Partial<Record<keyof Case, keyof CaseData>
   applicant2LegalProceedings: 'applicant2LegalProceedings',
   applicant2LegalProceedingsDetails: 'applicant2LegalProceedingsDetails',
   applyForFinancialOrder: 'applicant1FinancialOrder',
+  whoIsFinancialOrderFor: 'applicant1FinancialOrderFor',
```

Comment:
can't we change these to _applicant1WhoIsFinancialOrderFor_ and _applicant1WhoIsFinancialOrderFor_ for consistency now?